app/javascript/controllers/application.js

```diff
@@ -7,4 +7,4 @@ application.warnings = true;
 application.debug = false;
 window.Stimulus = application;
 
-export default { application };
+export { application };
```

Comment:
I ended up doing the same in my branch 😂 

Should definitely have gotten it out sooner, or made that change as a separate PR, so you didn't have to spend time debugging the same thing (like I suppose you did?(

Reply:
I actually just copied you on this. Remembered that you spoke of the generator :D